client/src/index.js

```diff
@@ -14,11 +14,13 @@ import TextField from '@material-ui/core/TextField';
 import Button from '@material-ui/core/Button';
 import Link from '@material-ui/core/Link';
 import Snackbar from '@material-ui/core/Snackbar';
+var util = require('util');
 
 import { login, signUp, getUID } from './firebase/firebaseAuth.js'
 var util = require('util');
```

Comment:
Can we fix these lines? I'm getting this error:
![image](https://user-images.githubusercontent.com/36905285/69988470-9145c300-14f6-11ea-958c-08f972cb25e0.png)